### ISSUE TITLE
devito setup.py installs top-level package just named 'examples' into site-packages

### ISSUE THREAD

**Issue by tjb900** [devitocodes/devito#1738]
I would argue this should at the very least be called `devito_examples` or something along those lines?
Same with `benchmarks`.

**Comment by mloubout** [comment]
Yes it is an issue. It is on the TODO list but we not sure how to proceed without completely breaking backward compatibility. It does need proper name change and/or restructuring of the repo.

**Comment by tjb900** [comment]
Ah, yeah -  backward compatibility...
One option to kick off the process might be to start raising a `DeprecationWarning`/`FutureWarning` when the modules are imported? Will happily submit a patch for that if you think it's a good idea.

**Comment by mloubout** [comment]
Yes would be the idea, just not sure how long making it a warning before moving it permanently.  In theory people who have `example` installed should still have it after the move but is a bit wishy.

**Comment by georgebisbas** [comment]
@mloubout was that related to the gempy notebook breakage?

**Comment by mloubout** [comment]
yes
